src/frontends/pytorch/src/op/repeat_interleave.cpp

```diff
@@ -68,8 +69,8 @@ OutputVector translate_repeat_interleave(const NodeContext& context) {
     } else {
         // repeats is not Constant or single element constant
         // Curently we support only case when repeats contains only one element. Otherwise next Reshape will fail.
-        auto repeats_input =
-            context.mark_node(std::make_shared<v1::Reshape>(context.get_input(1), const_1_list, false));
+        auto repeats_input = context.mark_node(std::make_shared<v1::ConvertLike>(context.get_input(1), const_1_list));
```

Comment:
Please use `Convert` instead of `ConvertLike`  because type of `const_1_list` is known

Reply:
Done